cockroachcloud/security-overview.md

```diff
@@ -5,93 +5,158 @@ toc: true
 docs_area: manage
 ---
 
-A {{ site.data.products.dedicated }} cluster is single-tenant (no shared machines) running in a Virtual Private Cloud (no shared network) and has data encryption-in-flight enabled by default. Additionally, {{ site.data.products.dedicated }} provides authentication, authorization, and SQL audit logging features to secure your clusters.
+This page summarizes the security features available in the two database cluster types offered by {{ site.data.products.db }}, **Serverless** and **Dedicated**.
 
-{{ site.data.products.serverless }} provides multi-tenant clusters running on GCP or AWS machines. They have similar encryption, authentication, and user authorization capabilities to {{ site.data.products.dedicated }}, but machines and networks are shared.
+A {{ site.data.products.serverless }} cluster is deployed on request for a specific customer in *shared* (multi-tenant) network and compute infrastrucutre.
+
+A {{ site.data.products.dedicated }} cluster is deployed on request for a specific customer in a cloud provider's network and compute infrastructure *dedicated* to that customer. This deployment may be distributed over multiple regions for added disaster-resilience. In addition to infrastructure isolation, dedicated clusters can be customized with advanced network, identity-management, and encryption-related security features required for high benchmark security goals such as PCI DSS compliance.
+
+Refer to [Payment Card Industry Data Security Standard (PCI DSS) Compliance in CockroachDB Dedicated](pci-dss.html)
```

Comment:
```suggestion
```

Reply:
Oh, I missed the second half of the suggested edit. I'd move the link up to "PCI DSS compliance" in the previous sentence.